src/views/LoginView.vue

```diff
@@ -1,114 +1,196 @@
 <template>
-  <v-container id="login-container" class="container">
-    <img id="logo" alt="Cuttle logo" src="/img/logo.png" />
-
-    <v-row>
-      <!-- Left side form -->
-      <v-col id="username-login-form" :cols="$vuetify.display.mdAndUp ? 6 : 12">
-        <h1 class="gradient-text">
-          {{ buttonText }}
-        </h1>
-
-        <form @submit.prevent="submitLogin">
-          <v-text-field
-            v-model="username"
-            variant="outlined"
-            :dense="$vuetify.display.mdAndDown ? true : false"
-            hint="Username"
-            data-cy="username"
-            autocomplete="username"
-            placeholder="Username"
-          />
-          <v-text-field
-            v-model="pw"
-            variant="outlined"
-            hint="Password"
-            :dense="$vuetify.display.mdAndDown ? true : false"
-            type="password"
-            data-cy="password"
-            :autocomplete="isLoggingIn ? 'current-password' : 'new-password'"
-            placeholder="Password"
-          />
-          <div id="login-button-container">
+  <div class="loginViewContainer">
+    <!--////////////////////////// 
+     /////Welcome Section ///// 
+    //////////////////////////-->
+    <section v-if="!isLoggingIn">
+      <v-container fluid id="welcome-container" class="welcomeContainer">
+        <nav class="d-flex justify-space-between align-center mb-2">
+          <img class="cardLogo" src="/img/loginView/logo-cards-behind.svg">
+          <v-btn  
+            variant="text"
+            class="text-h6"
+            @click ="$refs.usernameInput.focus()"
+          >
+            Sign up 
+            <img class="ml-2" src="/img/loginView/icon-account.svg" />
```

Comment:
We can replace this with the icon I mentioned above
```suggestion
            <v-icon icon="mdi-account-circle" color="white" class="ml-2"  />
```